test/libsolidity/semanticTests/isoltestTesting/bytecode.sol

```diff
@@ -0,0 +1,20 @@
+contract SmokeTest {
+    constructor()
+    {
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// constructor()
+// bytecode: 0x00112233 -> 0
+// bytecode: 0x60806040526 -> 1
+// bytecode: 0x806040526 -> 1
+// bytecode: 0x6040526 -> 1
+// bytecode: 0x5040526 -> 0
+// bytecode: "(60).*" -> 1
+// bytecode: ".*(40).*" -> 1
+// bytecode: ".*40.*6.*" -> 1
+// bytecode: ".*40.*600000.*" -> 0
+// bytecode: ".*40.*600011.*" -> 0
+// bytecode: ".*40.*6000.*" -> 1
```

Comment:
Hmm, couldn't we instead of true/false return the number of matches? That could be useful checking how many times something is used.

Reply:
If we do that, that's another reason to give it a more specific name. Otherwise it won't be easy to figure out what the number means without digging for that in the code.